### ISSUE TITLE
Publish linux/arm-64 images on docker hub

### ISSUE THREAD

**Issue by ionutpopa** [apache/drill#2950]
**Is your feature request related to a problem? Please describe.**
It's annoying that there is no other option but just linux/amd-64 on the docker hub, I think it would be really nice to have a image for linux/arm-64 

**Describe the solution you'd like**
I see that this was already discussed but no one bothered to do it: https://issues.apache.org/jira/browse/DRILL-8260
This is not an optimal process: having tickets opened that no one resolves.

**Comment by cgivre** [comment]
@ionutpopa 
Perhaps a little politeness might go a bit further.   Please understand that Drill is developed and maintained by volunteers who are not paid for their time.   If this is really important to you than you could either volunteer to assist, pay someone to do it, or simply ask nicely.

**Comment by tdunning** [comment]
Of course, @ionutpopa is correct. Running a project with all volunteers is not an optimal process in terms of getting everything done. Running it with an infinite budget close source development team would be much more "optimal".

But @cgivre is even more correct. Drill enables freedom. If you want it, feel free to make it so. It may be no more than a one line patch to the CICD scripts.

**Comment by ionutpopa** [comment]
Ok, I'll give a look on the CI/CD scripts later and what's there but first I'll probably search for a developer notice before doing anything.
Sorry for earlier, I get angry when I see old tickets.
@tdunning you're talking about the workflows folder inside .github, am I right?

**Comment by cgivre** [comment]
@ionutpopa No worries.  We welcome any help you can provide.

**Comment by tdunning** [comment]
@ionutpopa That is what I would have expected, but I don't see docker image packaging there.

There is likely another workflow that is part of cutting a release that generates the docker image. I have no idea where that might be. @cgivre should know more.

**Comment by jnturton** [comment]
@ionutpopa the Docker image builds themselves happen in Docker Hub but we have custom "hooks" in Drill source repo, see the file hooks/build. Let's try introducing something like `--platform linux/amd64,linux/arm64`.

**Comment by ionutpopa** [comment]
Hello everyone, I am curious why I am getting an error saying I don't have permissions to push branch to apache/drill. Can someone help with that regard?

**Comment by jnturton** [comment]
Only members of the committers group have that permission. If you create a fork and open a PR to main repo then I can help with branches or tags in the main repo that will trigger image builds on Docker Hub for testing. Once we've got an arm64 image we're happy with we can merge your PR into master. When you open the PR please use the provided template to allocate this work to DRILL-8260.

**Comment by ionutpopa** [comment]
Tout va bien, opened a PR.
